### PR TITLE
Add docs for new button

### PR DESCRIPTION
**Pull request by MelleD**
## Proposed change

Pause, resume and stop are published as a button entity

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/91923

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/home_connect.markdown [hunk 1]
```diff
@@ -96,6 +103,7 @@ Sets an option for the selected program.
 
 ### Service `home_connect.pause_program`
 
+This service is deprecated. Please use the button for this operation.
```
Comment (frenck) [inline source/_integrations/home_connect.markdown]: Please remove deprecated documentation instead of marking it deprecated.

Thanks 👍

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/home_connect.markdown (hunk 1 above)
Comment (MelleD) [reply source/_integrations/home_connect.markdown]: Even if the code is still there and exists?

Comment (MelleD) [reply source/_integrations/home_connect.markdown]: removed

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**